help_function/src/cub_blockexchange_tests.cpp

```diff
@@ -0,0 +1,148 @@
+// ====------ cub_blockexchange_tests.cpp---------- -*- C++ -* ----===////
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_extras/dpcpp_extensions.h>
+#include <sycl/ext/oneapi/group_local_memory.hpp>
+
+#include <stdio.h>
+#include <string.h>
+
+#define VALUES_PER_THREAD  4
+#define SIZE  16
+
+void init_data(int* data, int num) {
+  for(int i = 0; i < num; i++)
+    data[i] = i;
+}
+void verify_data(int* data, int num) {
+  return;

```

Comment:
We need to check function output is correct.

Reply:
Thanks I added this in the code.